content/docs/version-control-cicd/cicd/04.helpers.md

```diff
@@ -3,82 +3,106 @@ title: Helpers
-    .....
-    500 lines later
+  - id: t1
+    type: io.kestra.plugin.core.debug.Return
+    format: |
+      Lorem Ipsum is simply dummy text of the printing
+      .....
+      500 lines later
 ```
 
-You can replace the flow definition with this one:
+#### With helper
+
 ```yaml
 id: include
 namespace: company.team
 
 tasks:
-- id: t1
-  type: io.kestra.plugin.core.debug.Return
-  format: "[[> lorem.txt]]"
+  - id: t1
+    type: io.kestra.plugin.core.debug.Return
+    format: "[[> lorem.txt]]"
 ```
-And have a local file `lorem.txt` with the large content in it.
 
-The path can be:
-* `[[> lorem.txt]]`: a relative path from the flow (flow.yaml and lorem.txt are in the same directory),
-* `[[> /path/to/lorem.txt]]`: an absolute path,
-* `[[> path/to/lorem.txt]]`: a relative path from the flow (flow.yaml with a subdirectory `path/to/`).
+Then, create a local file named `lorem.txt` containing your text.
 
-When including a file, you must use the right YAML scalar type: literal (with or without quotes) for single-line scalars or folded for multiple-lines ones.
+---
 
+### Supported path formats
 
-:::alert{type="warning"}
-Includes are resolved recursively, so you can include a file from another include.
-This allows more complex things, but you need to take care that included files don't contain `[[ .. ]]`. If you need to have these characters in included files, escape them with `\[[ ...]]` !
-:::
+| Format | Description |
+|--------|--------------|
+| `[[> lorem.txt]]` | Relative path from the flow file (both in the same directory). |
+| `[[> /path/to/lorem.txt]]` | Absolute path. |
+| `[[> path/to/lorem.txt]]` | Relative path from the flow directory (e.g., `flow.yaml` in parent folder). |
 
-## Validate the flow to be uploaded to the server
+When including a file, ensure you use the correct YAML scalar type — literal (quoted or unquoted) for single-line values or folded for multiline content.
 
-There is a convenient command on the Kestra executable that allows validation of the current flow:
+::alert{type="warning"}
+Includes are **resolved recursively**, meaning included files can themselves contain additional includes.  
+Be careful: if your included file needs to display `[[ ... ]]` literally, escape it as `\[[ ... ]]`.
+::
+
+---
+
+## Local flow validation
+
+To validate your flow locally (especially if it uses helpers), use the `--local` flag.  
+This ensures validation runs with the same plugins as your local environment.
 
 ```bash
 ./kestra flow validate --local path-to-your-flow.yaml
 ```
-:::alert{type="info"}
-If your flow uses a helper function, flow validation must be done locally as the flow cannot be expanded on the webserver. Be careful that the local installation must have the same plugins as the remote installation.
-:::
 
+::alert{type="info"}
+Flows using helper functions **must** be validated locally since the expansion process cannot run on the Kestra webserver.
+::
```

Comment:
```suggestion
:::alert{type="info"}
Flows using helper functions **must** be validated locally since the expansion process cannot run on the Kestra webserver.
:::
```